cmat/output_generation/clinvar_to_evidence_strings.py

```diff
@@ -91,6 +91,14 @@ def collate_report(self):
             Total number of variant to consequence mappings\t{self.total_consequence_mappings}
                 Number of repeat expansion variants\t{self.repeat_expansion_variants}
                 Number of structural variants \t{self.structural_variants}'''.replace('\n' + ' ' * 12, '\n')
+        print(report)
+
+        # Confirm counts as expected, exit with error if not.
+        expected_total = clinvar_fatal + clinvar_skipped + clinvar_done
+        if expected_total != self.clinvar_total:
+            logger.error(f'ClinVar evidence string tallies do not add up to the total amount: '
+                         f'fatal + skipped + done = {expected_total}, total = {self.clinvar_total}')
+            sys.exit(1)
```

Comment:
Is the `exit` really necessary ? It only skips the final unmapped terms.
I guess exiting with non 0 makes sense but I'm wondering if we should do it at the end.

Reply:
That actually makes a lot more sense.